pkg/pillar/containerd/oci_test.go

```diff
@@ -849,43 +849,3 @@ func deepCopy(in interface{}) interface{} {
 	val = val.Elem()
 	return val.Interface()
 }
-
-func TestAllowAllDevicesInSpec(t *testing.T) {
```

Comment:
Would it make sense to have this test for no devices allowed?

Reply:
No, I just realized that and removed already, the PR (and the description) was updated.

Reply:
I've added the proper test again (to check for non-full device access).

Reply:
My question was whether the test should be kept but inverted now that you don't allow any devices by default. Or am I missing something?